packer/scripts/versions_to_json.rb

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env ruby
+
+require 'yaml'
+require 'json'
+
+def print_ansible_variable_json(version_hash)
```

Comment:
can we default to `nightly` that checks out develop and master? Would be nice to keep that as a default

Reply:
The default when running a normal packer build is nightly, you have to run via the special directions to get a non-nightly build.